apps/datahub/.eslintrc.json

```diff
@@ -21,7 +21,7 @@
           "error",
           {
             "type": "element",
-            "prefix": "datahub",
+            "prefix": ["datahub", "gn-ui"],
```

Comment:
Wondering, if this necessary or can be handled elsewhere. Without the "gn-ui" value, I get an eslint error when mocking a gn-ui component in a datahub spec.

Reply:
No this is not the way to go cause you could have component which starts with `gn-ui`
What I did in other tests is just ignore eslint for the line, might be the best way IMO

Reply:
Ok